cmd/embedded-cluster/install.go

```diff
@@ -638,6 +649,15 @@ func validateAdminConsolePassword(password, passwordCheck string) bool {
 	return true
 }
 
+// return only the first private CA path for now - troubleshoot needs a refactor to support multiple CAs in the future
+func getPrivateCAPath(c *cli.Context) string {
+	privateCA := ""
+	if len(c.StringSlice("private-ca")) > 0 {
+		privateCA = c.StringSlice("private-ca")[0]
+	}
+	return privateCA
+}
```

Comment:
This won't work on joins - I think you'll need to add the CAs to the preflight inputs